packages/react-storage/src/components/StorageBrowser/composables/ActionDestination.tsx

```diff
@@ -23,26 +23,26 @@ export const ActionDestination = ({
   items,
   label,
 }: ActionDestinationProps): React.JSX.Element | null => {
+  const Resolved = useResolvedComposable(Navigation, 'Navigation');
```

Comment:
Instead of doing this at this level - think we should resolve the navigation component in the ActionDestinationControl and feed it to this component via a `navigableDestination` prop for consistency (like what we're doing with the loading indicator)

cc @calebpollman wdyt